test/integration/sequelize.test.js

```diff
@@ -490,11 +449,11 @@ describe(Support.getTestDialectTeaser('Sequelize'), function() {
             foos: 'bars'
           }, { transaction: t });
         }).then(function() {
-          return this.sequelize.query('SELECT @foo as `foo`, @foos as `foos`', null, { raw: true, plain: true, transaction: this.t });
+          return this.sequelize.query('SELECT @foo as `foo`, @foos as `foos`', { transaction: this.t, type: this.sequelize.QueryTypes.SELECT });
```

Comment:
Hmm yeah it might  be good to default to `SELECT` when plain is true...

My reasoning was that plain was never a documented option, so I wanted to remove it completely ;)